test/unit/pdf_find_controller_spec.js

```diff
@@ -1022,4 +1022,36 @@ describe("pdf_find_controller", function () {
       pageMatchesLength: [[1, 1, 1, 1, 1, 1, 1, 1, 1]],
     });
   });
+
+  it("dispatches updateFindControlState with picked state properties", async function () {
```

Comment:
Nit: Since the actual event-name is lowercase let's be consistent here, and we can slightly shorten the test description as well.
```suggestion
  it("dispatches updatefindcontrolstate with correct properties", async function () {
```